requirements.txt

```diff
@@ -1,9 +1,11 @@
 django==1.8.2
-scipy==0.13.3
-matplotlib==1.3.1
-PyBrain==0.3
-psycopg2==2.6.1
-django-chartit==0.1
-simplejson==3.8.2
-Cython==0.23.5
-sklearn==0.0
+scipy
```

Comment:
I think the problem while building was that that version of scipy was not possible to be built with the newest numpy (and I didn't know which version of numpy you had, because it's not in the requirements). So I just dropped all the versions (except for Django because that broke down the startup) and build it. Now, I think I found the proper version of numpy so let's see.

It's not instant to see if the things work out, because install of numpy and then scipy takes some time (~20 miuntes on my crappy laptop).